README.md

```diff
@@ -62,7 +62,7 @@ After the docker-compose built is ready, you need to configure the Django app an
 Log in to the Django container to apply the migrations. From your terminal, run this command.
 
 ```bash
-docker exec -it Django bash
+docker exec -it django bash
```

Comment:
I tried Django with capital 'D' and it did not work, I guess should be 'django' as this is the name of the container; but maybe I am missing something.